packages/kit/src/core/sync/write_tsconfig.js

```diff
@@ -4,23 +4,43 @@ import colors from 'kleur';
 import { posixify } from '../../utils/filesystem.js';
 import { write_if_changed } from './utils.js';
 
-/** @param {string} file */
-const exists = (file) => fs.existsSync(file) && file;
+/**
+ * @param {string} cwd
+ * @param {string} file
+ */
+function maybe_file(cwd, file) {
```

Comment:
I stumbled over this, too. It's short form for `if (fileexists) return file`. Since this isn't part of the resulting app it's not sensitive to a few extra characters and that is more readable, I'm going to change this 👍